packages/eui/src/components/code/code_block.test.tsx

```diff
@@ -46,6 +46,19 @@ describe('EuiCodeBlock', () => {
       });
     });
 
+    describe('copyAriaLabel', () => {
+      it('is rendered', () => {
```

Comment:
another minor testing nit/foible of mine - i don't love `describe()` blocks with just one test inside them, I find that pretty meaningless. I'd prefer to pare this down to just:

```suggestion
    test('copyAriaLabel', () => {
```

or, if you prefer `it()` syntax/grammar:

```suggestion
    it('renders `copyAriaLabel` on the copy button', () => {
```

personally I use `it()` for descriptive sentences and `test()` for single nouns, especially for list type tests where it's just a bunch of similar X tests in a row. (e.g. `describe('colors'), test('blue'),  test('red'), test('green')`)

Reply:
Totally agree! With the nit-pick and the `it` vs `test` insight (more or less). Generally, I tend to use `it` most often because it pairs nicely with the behavior-driven type of testing, e.g. "it should open a modal". I don't like `test('blue')` approach though because it's missing the expected behavior / state - many times it's implicit or I have to read the test case to understand the desired outcome.

And here again, instead of going with my instinct I went with consistency. In the next PRs I'll try to be a bit more proactive on code and testing best practices.

While at it, I'll refactor this whole test suite cause all the other cases are 1 test case per describe as well 🙈

Reply:
@cee-chen now that I think about it, let me separate those 2 things so that we can merge the actual feature already and I'll open a second PR with test suite refactor. I'll just apply your suggestions to the `copyAriaLabel` prop and that's it.